models/yang/sonic/sonic-portchannel.yang

```diff
@@ -0,0 +1,77 @@
+module sonic-portchannel {
```

Comment:
Please fix the indentation in this file to 4 spaces per tab

Reply:
Done!